pages/[[...swingset]].jsx

```diff
@@ -128,6 +130,13 @@ const components = {
   ComboboxField, // @TODO - Consider Swingset support for components at nested entry points
 }
 
-export default createPage({ components, logo: <Logo />, index: <Index /> })
+export default function SwingsetPage(props) {
+  return createPage({ components, logo: <Logo />, index: <Index /> })(props)
+}
+
 export const getStaticPaths = createStaticPaths()
-export const getStaticProps = createStaticProps({ components })
+export const getStaticProps = async (ctx) => {
+  let staticProps = await createStaticProps({ components })(ctx)
+  staticProps = loadSampleMdx(ctx, staticProps)
+  return staticProps
+}
```

Comment:
@BRKalow this is my approach for rendering a "real" mdx file for the `react-components` `docs-page` preview

- see `swingset-extensions/load-sample-mdx/index.ts`